parsing/cLexer.ml4

```diff
@@ -513,20 +513,27 @@ let process_sequence loc bp c cs =
   aux 1 cs
 
 (* Must be a special token *)
-let process_chars loc bp c cs =
+let process_chars diff_mode loc bp c cs =
```

Comment:
While rebasing to deal with conflicts could you also make it a labelled argument? Thanks.